google_compute_engine/accounts/accounts_daemon.py

```diff
@@ -23,11 +23,12 @@
 
 from google_compute_engine import config_manager
 from google_compute_engine import file_utils
+from google_compute_engine import LOCALSTATEDIR
 from google_compute_engine import logger
 from google_compute_engine import metadata_watcher
 from google_compute_engine.accounts import accounts_utils
 
-LOCKFILE = '/var/lock/google_accounts.lock'
+LOCKFILE = LOCALSTATEDIR+'/lock/google_accounts.lock'
```

Comment:
Please add a space around both sides of the `+`. (Apply this comment everywhere).